contrib/scripts/kind.sh

```diff
@@ -34,12 +34,12 @@ if ! have_kind; then
     echo "  https://kind.sigs.k8s.io/docs/user/quick-start/#installation"
 fi
 
-if [[ "${#}" -gt 5 ]]; then
+if [[ "${#}" -gt 6 ]]; then
```

Comment:
nitpicking: `$#` is always a number, no need to stringify it. Also POSIX `test(1)` is enough here (but `[[` for consistency is ok too):
```suggestion
if [ $# -gt 6 ]; then
```